vars/DevelopPipeline.groovy

```diff
@@ -1,81 +1,94 @@
 import org.lsst.ts.jenkins.components.Csc
 
-def call(name, idl_name) {
+def call(name, idl_name, module_name, extra_packages=[]) {
```

Comment:
I think in order to support the `extra_packages`, assuming these do no ship with the container we may have to do something similar to what is done in the MTAOS and ts_externalscripts. 

Basically, use the option `skipDefaultCheckout`, then create a "Cloning Repos" stage where we do something like [this](https://github.com/lsst-ts/ts_externalscripts/blob/4f384f4cc689b2a65119a675757cd7c4441cfac9/Jenkinsfile#L37-L52).

Just something to keep in mind. We should probably leave that for a future iteration of the library.